odl/contrib/shearlab/shearlab_operator.py

```diff
@@ -171,8 +179,50 @@ def _call(self, x):
 
             @property
             def adjoint(self):
-                """The adjoint operator."""
-                return ShearlabOperatorAdjointInverse()
+                """The inverse operator."""
+                op = self
+
+                class ShearlabOperatorInverseAdjoint(odl.Operator):
+
+                    """
+                    Adjoint of the inverse/Inverse of the adjoint
+                    of shearlet transform.
+
+                    See Also
+                    --------
+                    odl.contrib.shearlab.ShearlabOperator
+                    """
+
+                    def __init__(self):
+                        """Initialize a new instance.
+                        Parameters
```

Comment:
This `__init__` method seems to take no parameters as input. So the docstring could just be `"""Initialize a new instance."""`. This also goes for the methods on [line 78](https://github.com/arsenal9971/odl/blob/5e762cb9ed7cb406591bb1adae5496f2b62339f2/odl/contrib/shearlab/shearlab_operator.py#L78), [line 117](https://github.com/arsenal9971/odl/blob/5e762cb9ed7cb406591bb1adae5496f2b62339f2/odl/contrib/shearlab/shearlab_operator.py#L117), and [line 164](https://github.com/arsenal9971/odl/blob/5e762cb9ed7cb406591bb1adae5496f2b62339f2/odl/contrib/shearlab/shearlab_operator.py#L164).

Reply:
Right, let me commit tomorrow this fixes, sorry for my lack proficiency on git, I will cherry pick tomorrow

Reply:
No worries. Thank you for great contributions to `odl`! :smile:

Reply:
Remaining TODO